### PR TITLE
Separate source scss out of rails paths

### LINKED ISSUE #10: referencing gem via git triggers missing bootstrap dependency
If you put this in your Gemfile:

``` ruby
gem 'flipflop', git: 'https://github.com/voormedia/flipflop.git'
```

And you don't include bootstrap 4, then you will get an error like:

```
Sass::SyntaxError at /

File to import not found or unreadable: bootstrap/reboot.
```

### PR DESCRIPTION
**Pull request by jcoyne**
This prevents Rails from compiling the assets when the gem is loaded
from a git link.

Fixes #10

### CODE REVIEW

**Comment by rolftimmermans** [comment]
The idea was that all development versions use the SCSS, but for released versions we compile to CSS and remove the `bootstrap` dependency.

The problem is that the development version never requires `bootstrap` anywhere.

I'll leave this open and will give it some thought. In the mean time, add this to your application somewhere if you use Flipflop from git:

`require "bootstrap"`

**Comment by jcoyne** [comment]
I understand what you are trying to do, and I think now I have this PR at a place where it does exactly what it did before  + it works if you required it via `gem 'flipflop', git: '...'`

File: .travis.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ notifications:
   email: false
   slack:
     secure: crOO1QGnFn9T1DpVgxkukTSiN8lQq09X8WF8oi1Eoa7Liex4gzWq7f8wlIXPrRFAsNjU47fSrz+L1C6Eg068Vd2df9pkqjW0pLNeqUViJ35TzpYISTtzJrX+x3nvCQLlvS4leP6lkijlsvlu1IN0xnXadW5cmcMoEcPo4Yma1RUklwlreSRIEmJiutYKVFRw3gIZA9vsnXNEcD408mvSY/8Kuw+hmRQupODUalXDpZo1q3HH+ZPQq+/rGuJ7XRf9sBtxjpUF0G4FJZQhVP4CrLNYVBE/83rHJ6HSf6u3SlYVIMiautq0nWpVLPHUrkOPJVeVh6EPtoFeI/cehH1NyoAVvL5a39wFRBlJ4jVPWUrrnihJT/6+P6GM9PSnYogxtIoTsdrYES2FgtWGgwG5uLyw8U6bW7G7rCzQwBP7enVHWVCbDgdSSjE1Mg1I9qhRuL6pHs5des4VKk6pfD3p+BRqLmOZR2jx4v8MFwakSFqQWOMxaD0U1lfxecqSx9OkwWEhCFSnHeXeHInEhY6qKCdZZzT+beYn0xppUMPJGMTqe5+po8gL+5MxQwI8Xs/5hSve5frfmuS7UQf5BnFMOzwoThQrXCFRz58wXvcZTD9eTdVBV44Hsi5OLdYn9K58sNUhgcxGfRgdE7Gy9P7DD4fHPGakD/Tz++HuCmCUXNs=
-before_deploy: rake assets:compile
+before_deploy: rake assets:compile --trace
```
Comment (mjgiarlo) [inline .travis.yml:39]: Do you still need this, @jcoyne?

**Comment by mjgiarlo** [comment]
@rolftimmermans Since @jcoyne has preserved past behavior, are you inclined to reconsider this PR?

**Comment by mjgiarlo** [comment]
Hi, @rolftimmermans. Are you still maintaining this gem? Wondering about :arrow_up: and https://github.com/voormedia/flipflop/pull/14#issuecomment-252139905. Thank you.

**Comment by rolftimmermans** [comment]
Definitely maintained, but this is a tricky issue that I haven't had a chance yet to fully look into. I think that when we solve #15 this is going to be solved also.

**Comment by mjgiarlo** [comment]
Thanks, @rolftimmermans.

**Comment by rolftimmermans** [comment]
Closed by 2f7860e based on your contribution. Thanks!

**Comment by jcoyne** [comment]
Thanks!

**Pull request closed without merging**